modules/claims_api/spec/requests/v0/claims_request_spec.rb

```diff
@@ -96,25 +107,56 @@
       end
     end
 
-    it 'shows a single Claim through auto established claims', run_at: 'Wed, 13 Dec 2017 03:28:23 GMT' do
-      create(:auto_established_claim,
-             auth_headers: { some: 'data' },
-             evss_id: 600_118_851,
-             id: 'd5536c5c-0465-4038-a368-1a9d9daf65c9')
+    it 'shows a single Claim when camel-inflected', run_at: 'Wed, 13 Dec 2017 03:28:23 GMT' do
       VCR.use_cassette('evss/claims/claim') do
-        get(
-          '/services/claims/v0/claims/d5536c5c-0465-4038-a368-1a9d9daf65c9',
-          params: nil,
-          headers: {
-            'X-VA-SSN' => '796043735', 'X-VA-First-Name' => 'WESLEY',
-            'X-VA-Last-Name' => 'FORD', 'X-VA-EDIPI' => '1007697216',
-            'X-Consumer-Username' => 'TestConsumer', 'X-VA-User' => 'adhoc.test.user',
-            'X-VA-Birth-Date' => '1986-05-06T00:00:00+00:00', 'X-VA-LOA' => '3'
-          }
-        )
-        expect(response).to match_response_schema('claims_api/claim')
+        get '/services/claims/v0/claims/600118851',
+            params: nil,
+            headers: request_headers_camel
+        expect(response).to match_camelized_response_schema('claims_api/claim')
+      end
+    end
+
+    context 'with an auto established claim' do
```

Comment:
created this context and refactored a little to DRY-up the test setup for these two

Reply:
Thank you for this comment! It made reviewing 1000x easier, knowing what to look for.